package.json

```diff
@@ -95,7 +102,16 @@
     "postinstall": "npm run build",
     "update-codegen": "node scripts/update-codegen.js",
     "generate-client": "node scripts/generate-client.js",
-    "postgenerate-client": "node node_modules/webpack/bin/webpack.js"
+    "postgenerate-client": "node node_modules/webpack/bin/webpack.js",
+    "lintfix": "jscs --fix src/**/*.js test/**/*.js",
+    "postlint": "jshint src/**/*.js test/**/*.js",
```

Comment:
per your findings, this globbing doesn't always work.  :(  i think a `shopt` glob option must be set?  anyway, a week or so ago i published a version of coins-validate that drops the `/**/*.js` which our libs still successfully recurse!  :game_die: :game_die: :game_die: :game_die: :game_die: